### PR TITLE
Configure templates module

### PR DESCRIPTION
**Pull request by stbenjam**
Really, http://projects.theforeman.org/issues/8990 would be nice for this as well, but not sure I'll have time to take care of that one.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -105,6 +105,11 @@
 #                               try to determine this automatically.
 #                               type:boolean
 #
+# $templates::                  Use Templates
```
Comment (domcleal) [inline manifests/init.pp]: Enable template proxying feature, or something?

File: manifests/params.pp [hunk 2]
```diff
@@ -79,6 +79,10 @@
   $puppetssh_wait     = false
   $puppet_user        = 'root'
 
+  # Template settings
+  $templates          = false
+  $template_url       = "http://${::fqdn}:${port}"
```
Comment (domcleal) [inline manifests/params.pp]: isn't this meant to be Foreman's port?

File: manifests/init.pp [hunk 3]
```diff
@@ -105,6 +105,11 @@
...
+#                               type:boolean
+#
+# $template_url::               URL a host should use for provisioning templates
```
Comment (ekohl) [inline manifests/init.pp]: Host might be ambigous in the content of a proxy. Maybe explicitly use client if you mean hosts that are provisioning?

File: manifests/init.pp (hunk 1 above)
Comment (stbenjam) [inline manifests/init.pp]: The convention everywhere else is `Use $feature`

Comment (domcleal) [inline manifests/init.pp]: I don't think we should stick to it, it's a terrible description :)

**Comment by domcleal** [comment]
:+1: apart from the parameter descriptions.

File: manifests/init.pp (hunk 1 above)
Comment (ekohl) [inline manifests/init.pp]: I agree; even ```Use Templates feature``` would be better.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**